### DISCUSSION TITLE
vscode and preview

### DISCUSSION THREAD

**Discussion by JMueller-edu** [LiaScript/LiaScript#166] [Q&A] [answered]
Hallo Leute,

funktioniert bei euch in Vscode die LiaScript-Preview Erweiterung normal?

Bei mir funktioniert die Vorschau unter Window 11 im Edge Browser (Version 122.0.2365.80 (Offizielles Build) (64-Bit)) nur wenn die Markdown-Datei im Onedrive Ordner liegt. Bei einem lokalen Pfad von C: erhalte ich immer die angehängte Fehlermeldung.

![image](https://github.com/LiaScript/LiaScript/assets/105539256/31eafd2d-6dac-4ec1-ba47-8863664137df)

Inhalt der Markdowndatein (Onedrive und C:) ist der lia-init Kontext. Weiterhin wird die Datei auf C: teil eines Github-Repository, während die andere wie gesagt im Onedrive Ordner liegt. 
 
Hat jemand eine Idee was ich falsch mache?

Mit besten Grüßen
Jan

**Comment by andre-dietrich** [comment, marked as answer]
Hallo Jan,

sowas kann passieren, wenn man nur die Datei öffnet. VSCode sagt dem Plugin dann nicht, wo das Projektverzeichnis liegt. Kannst du nochmal folgendes probieren:

File >> Open Folder >> Und das Verzeichnis öffnen, in dem deine MD-Datei liegt

Wenn es das löst, dann müssen wir nochmal über eine entsprechende Fehlermeldung nachdenken :see_no_evil: 

 LG André

**Reply by JMueller-edu** [reply]
Hi André,

vielen Dank für die schnelle und aufschlussreiche Antwort.

Tatsächlich scheint es an meinem Multiroot-Workspace zu liegen.

Ich habe wie du vorgeschlagen hast, dass Verzeichnis mit der MD-Datei einzeln geöffnet und siehe da das Live-Preview geht.

Anschließend habe ich meine anderen Ordner dem Workspace wieder hinzugefügt. Für den zu erst hinzugefügten Ordner funktioniert das Preview-Plugin weiterhin, für alle anderen nicht.

Mit freundlichen Grüßen
Jan
